aws/cloudformation/standalone/gen_ai_curriculum/gen_ai_curriculum.yml.erb

```diff
@@ -0,0 +1,82 @@
+# Gen AI Curriculum Resources
+# This template is manually deployed to create/update Gen AI Curriculum resources.
+# TODO:
+  # Fix hardcoded execution role ARN
+
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: SageMaker resources for Gen AI Curriculum.
+
+Parameters:
+  Environment:
+    Type: String
+    Default: "test"
+    Description: "Environment being deployed to (prod or test)"
+  HuggingFaceToken:
+    Type: String
+    Description: "HuggingFace token required for deploying HuggingFace models."
```

Comment:
Might be worth noting if this needs to be a write token or if a read token will do (or even more fine-grained? no idea if that's a thing).